scripts/docstrings_checker.py

```diff
@@ -153,6 +155,16 @@ def possible_exc_types(node):
 
 
 def docstringify(docstring):
+    """Converts a docstring to its Docstring object as defined in the
+    pylint library.
+
+    Args:
+        docstring: Docstring for a particular class or function
```

Comment:
Missing type into. Please note that this type should be same as what you get after executing `type(docstring)` in Python. Also, add full-stop at the end of the line.

Reply:
@vibhor98 done!